recipes/libspnav/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "libspnav" %}
```

Comment:
It is fine to omit the `name` jinja variable and replace all `{{ name }}` with `libspanv` because `name=libspanv` is constant for the recipe.